src/App.js

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from "react";
-
+import AppRoutes from "./routes";
 class App extends Component {
 	render() {
 		return (
-			<div><h1>Authors Haven </h1></div>
+			<AppRoutes/>
 		);
```

Comment:
Use 2 spaces in your code editor to improve your indentation and code outlook, look at the indentation used in `package.json`for reference

Reply:
Noted

Reply:
@LehruAngela eslint was configured to use a tab which are 4 spaces for indetation. should I change that rule in eslintrc.js?

Reply:
@kimbugp is removing that rule. Then you will revert your work.

Reply:
@LehruAngela I have made the change